### DISCUSSION TITLE
Support for list-style-image

### DISCUSSION THREAD

**Discussion by MacTavish-69** [tailwindlabs/tailwindcss#3232] [Ideas]
Currently, there is no support for "list-style-image", although it is less used but sometimes it is needed so it feels a missing bit in our lovely tailwind css!

**Comment by ShivamJoker** [comment]
No updates wow ?

**Comment by ktisakib** [comment]
is it yet aadded

**Reply by solo-smart** [reply]
No

**Comment by RandyGrullon** [comment]
nothing yet?

**Comment by smartdev322** [comment]
Still not?

**Comment by Mukaba** [comment]
is there any update?

**Comment by cytrobit** [comment]
Well, I found a solution - you can input the svg code just after `<li>` tag and before that `<li>` content. It works for me.
![tailwind-list-style-image](https://user-images.githubusercontent.com/82907373/225763071-3e99d34a-7676-4a6a-8e45-3f7383d694b3.jpg)

**Reply by raymondl5490** [reply]
Really, Let me check, I hope it will work.

**Comment by adamwathan** [comment]
Planning to merge support for this tomorrow 🍻
